libs/utils/utils.js

```diff
@@ -483,15 +483,23 @@ export function decorateImageLinks(el) {
   const images = el.querySelectorAll('img[alt*="|"]');
   if (!images.length) return;
   [...images].forEach((img) => {
-    const [source, alt] = img.alt.split('|');
+    const [source, alt, playBtn] = img.alt.split('|');
     try {
       const url = new URL(source.trim());
       if (alt?.trim().length) img.alt = alt.trim();
       const pic = img.closest('picture');
       const picParent = pic.parentElement;
```

Comment:
```suggestion
      const parent = img.closest('picture')?.parentElement;
```
If you're not doing any checks on picture, I would probably combine the two lines.

Reply:
We are using the pic and picParent elements later in the function to find the location for aTag, moving the image in aTag and is also passed as a param to the modal play button. Hence we have 2 variables to store this information.